src/main/scala/org/broadinstitute/dsde/firecloud/service/ShareLogService.scala

```diff
@@ -0,0 +1,52 @@
+package org.broadinstitute.dsde.firecloud.service
+
+import akka.pattern.pipe
+import akka.actor.{Actor, Props}
+import org.broadinstitute.dsde.firecloud.Application
+import org.broadinstitute.dsde.firecloud.model.ModelJsonProtocol.impShareFormat
+import org.broadinstitute.dsde.firecloud.dataaccess.ShareLogDAO
+import org.broadinstitute.dsde.firecloud.model.ShareLog.ShareType
+import org.broadinstitute.dsde.firecloud.model.UserInfo
+import org.broadinstitute.dsde.firecloud.service.PerRequest.{PerRequestMessage, RequestComplete}
+import org.broadinstitute.dsde.firecloud.service.ShareLogService._
+import org.broadinstitute.dsde.rawls.model.{AttributeFormat, PlainArrayAttributeListSerializer}
+import spray.httpx.SprayJsonSupport
+import spray.json.DefaultJsonProtocol._
+
+import scala.concurrent.{ExecutionContext, Future}
+
+object ShareLogService {
+  sealed trait ShareLogMessage
+  case class LogShare(userId: String, sharee: String, shareType: ShareType.Value) extends ShareLogMessage
+  case class GetShares(userId: String, shareType: Option[ShareType.Value] = None) extends ShareLogMessage
+  case class GetSharees(userId: String, shareType: Option[ShareType.Value] = None) extends ShareLogMessage
```

Comment:
it looks like the `GetShares` case class, and the `getShares` method, are never used. Clean them up?